### PR TITLE
kelly/checkbox_for_TIN

### PR DESCRIPTION
**Pull request by kellybinary**

### CODE REVIEW

File: src/templates/_common/components/forms_common_rows.jsx [hunk 1]
```diff
@@ -169,3 +169,19 @@ export const ClientMessage = () => (
...
+    <div id='tax_id_terms'>
+        <div className='gr-12 gr-padding-10'>
+            <input type='checkbox' id='chk_tax_id' />
+            <label htmlFor='chk_tax_id'>
+                {it.L('I hereby declare and confirm that the above given information is true and complete. Moreover, I undertake to inform Binary Investments (Europe) Ltd. about any changes therein without undue delay. Information can be edited by accessing your profile.')}
+            </label>
+        </div>
+        <div className='gr-12 gr-padding-10'>
+            <p className='no-margin'>
+                <i><span className='errorfield'>* </span>{it.L('Please note that you might be considered resident for tax purposes in more than one jurisdictions. [_1] does not take any responsibility for providing you with advice or information on how to determine your tax residence. If you are in any doubt regarding your tax residence kindly consult your tax advisor and make sure you provide us with the correct information.', it.website_name)}</i>
```
Comment (negarn) [inline src/templates/_common/components/forms_common_rows.jsx]: should it be `considered **a** resident`?

File: src/templates/app/new_account/financial.jsx [hunk 2]
```diff
@@ -40,6 +41,12 @@ const Financial = () => (
             </Fieldset>
 
             <Fieldset legend={it.L('Tax Information')}>
+                <div className='gr-12'>
+                    <p>{it.L('As a legal and regulatory requirement BIEL is obliged to collect certain information relating to CRS/FATCA compliance. We therefore kindly ask you to fill in the provided self-certification form before we can enable any further activity on your account, including closing of the account.')}</p>
```
Comment (jy-binary) [inline src/templates/app/new_account/financial.jsx]: replace BIEL with Binary Investments (Europe) Ltd.

File: src/templates/app/new_account/financial.jsx [hunk 3]
```diff
@@ -40,6 +41,12 @@ const Financial = () => (
...
+                    <p>{it.L('The information provided by you may only be disclosed to the authorities legally charged with collecting this information for the purposes of CRS/FATCA reporting and only to the extend, to which BIEL is legally obliged to collect and disclose it. The information shall not be used, disclosed or processed in any other way at any time.')}</p>
```
Comment (jy-binary) [inline src/templates/app/new_account/financial.jsx]: replace BIEL with Binary Investments (Europe) Ltd.

File: src/templates/app/new_account/financial.jsx (hunk 2 above)
Comment (jy-binary) [reply src/templates/app/new_account/financial.jsx]: remove mention of closing account!!

File: src/templates/app/new_account/financial.jsx (hunk 3 above)
Comment (jy-binary) [reply src/templates/app/new_account/financial.jsx]: improve terrible wording

File: src/templates/app/user/settings/personal_details.jsx [hunk 4]
```diff
@@ -75,7 +76,12 @@ const PersonalDetails = () => (
                 </FormRow>
             </Fieldset>
 
-            <Fieldset className='invisible RealAcc ja-hide' legend={it.L('Tax Information')}>
+            <Fieldset id='fieldset_tax_information' className='invisible RealAcc ja-hide' legend={it.L('Tax Information')}>
+                <div id='tax_information_note' className='gr-12 invisible'>
+                    <p>{it.L('As a legal and regulatory requirement BIEL is obliged to collect certain information relating to CRS/FATCA compliance. We therefore kindly ask you to fill in the provided self-certification form before we can enable any further activity on your account, including closing of the account.')}</p>
```
Comment (jy-binary) [inline src/templates/app/user/settings/personal_details.jsx]: why is this duplicated text??

**Pull request merged**